test/Histogram.hh

```diff
@@ -0,0 +1,49 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file Histogram.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <vector>
+
+#include "corecel/Types.hh"
+#include "corecel/cont/Array.hh"
+
+namespace celeritas
+{
+namespace test
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate the histogram of a set of values.
+ */
+class Histogram
```

Comment:
Maybe this class should use `UniformGrid` as an implementation detail? (Then it could be easily templated to allow log/nonuniform grids?) Should the input be a vector by reference (resized during intialization, or with its size used instead of an input number of bins) so this just becomes an accumulator? Should there be a helper function that turns normalizes the result into fractions rather than counts? I guess there's a lot of ways to take this in terms of flexibility vs simplicity... so maybe best to leave as-is?

Reply:
Yeah, I considered making it more like a numpy histogram (construct with a vector of bin edges, have a density option) but went for simple because that's all we needed currently. Maybe I'll add the density helper, and if we decide to make this class more flexible later on we can...